application/src/test/java/com/ericsson/bss/cassandra/ecchronos/application/config/TestConfig.java

```diff
@@ -166,6 +166,68 @@ public void testDefault() throws Exception
         assertThat(restServerConfig.getPort()).isEqualTo(8080);
     }
 
+    @Test
+    public void testDefault() throws Exception
+    {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        File file = new File(classLoader.getResource("nothing_set.yml").getFile());
+
+        ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+
+        Config config = objectMapper.readValue(file, Config.class);
+
+        Config.ConnectionConfig connection = config.getConnectionConfig();
+
+        Config.NativeConnection nativeConnection = connection.getCql();
+        assertThat(nativeConnection.getHost()).isEqualTo("localhost");
+        assertThat(nativeConnection.getPort()).isEqualTo(9042);
+        assertThat(nativeConnection.getTimeout().getConnectionTimeout(TimeUnit.MILLISECONDS)).isEqualTo(0);
+        assertThat(nativeConnection.getProviderClass()).isEqualTo(DefaultNativeConnectionProvider.class);
+        assertThat(nativeConnection.getDecoratorClass()).isEqualTo(NoopStatementDecorator.class);
+
+        Config.Connection jmxConnection = connection.getJmx();
+        assertThat(jmxConnection.getHost()).isEqualTo("localhost");
+        assertThat(jmxConnection.getPort()).isEqualTo(7199);
+        assertThat(jmxConnection.getProviderClass()).isEqualTo(DefaultJmxConnectionProvider.class);
+
+        RepairConfiguration expectedConfiguration = RepairConfiguration.newBuilder()
+                .withRepairInterval(7, TimeUnit.DAYS)
+                .withParallelism(RepairOptions.RepairParallelism.PARALLEL)
+                .withRepairWarningTime(8, TimeUnit.DAYS)
+                .withRepairErrorTime(10, TimeUnit.DAYS)
+                .withRepairUnwindRatio(0.0d)
+                .withTargetRepairSizeInBytes(Long.MAX_VALUE)
```

Comment:
Nit:

 This might have been my fault using Long.MAX_VALUE earlier, this should actually point to RepairConfiguration.FULL_REPAIR_SIZE.

Could you update both test methods?